frontend/packages/shared/src/components/CookieBanner/CookieBanner.tsx

```diff
@@ -42,25 +42,29 @@ export const CookieBanner: FC<PropsWithChildren<CookieBannerProps>> = ({
     setOpen(false);
   };
 
+  if (!open) return null;
+
   return (
```

Comment:
```suggestion
  return open && (
```

Ei merkitystä toiminnallisuuteen mutta tää ois ehkä enemmän se tyyli mitä Reactin kans käytetty.